src/query/execute_schema.c

```diff
@@ -8853,6 +8854,24 @@ execute_create_select_query (PARSER_CONTEXT * parser, const char *const class_na
   return error;
 }
 
+/*
+ * has_replication_key_constraint() -
+ *   return : true if the class has a UNIQUE constraint
+ *   class_obj(in): class object pointer
+ */
+bool
+has_replication_key_constraint (DB_OBJECT * class_obj)
```

Comment:
상위 수준에서는 간단한 함수명이 더 가독성이 좋을 것 같습니다.
```suggestion
has_replication_key (DB_OBJECT * class_obj)
```

Reply:
@hornetmj 희수님 코멘트에따라 리팩토링해보니 has_replication_key_constraint() 함수를 호출하지 않고 execute에서 바로 classobj_has_class_repl_key_constraint() 함수를 호출이 가능해 해당 함수를 제거하였습니다.